api/config.yaml

```diff
@@ -227,3 +227,26 @@ yarnaudit:
   language: JavaScript
   default: true
   timeOutInSeconds: 360
+
+gitleaks:
+  name: gitleaks
+  image: huskyci/gitleaks
+  imageTag: "2.1.0"
+  cmd: |+
+    mkdir -p ~/.ssh &&
+    echo 'GIT_PRIVATE_SSH_KEY' > ~/.ssh/huskyci_id_rsa &&
+    chmod 600 ~/.ssh/huskyci_id_rsa &&
+    echo "IdentityFile ~/.ssh/huskyci_id_rsa" >> /etc/ssh/ssh_config &&
+    echo "StrictHostKeyChecking no" >> /etc/ssh/ssh_config &&
+    cd src
```

Comment:
Hey, @abzcoding! 🙃 This `cd src` is out of context, I guess. This command is returning an error inside the container: 
```
/ # docker logs priceless_jennings
/bin/sh: cd: line 6: can't cd to src
```
Would you mind removing it, please?

Reply:
yep it seams copy/paste is not a good idea after all :)